core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -102,14 +99,15 @@ object ConsumerGroupCommand {
     if (!useNewConsumer) {
       val topics = zkUtils.getTopicsByConsumerGroup(group)
       if (topics.isEmpty) {
-        warnNoTopicsForGroupFound
+        println("No topic available for consumer group provided")
       }
       topics.foreach(topic => describeTopic(zkUtils, group, topic, channelSocketTimeoutMs, channelRetryBackoffMs, opts))
```

Comment:
Doesn't make much difference, but this might be a little cleaner as an "else" block.